test/e2e/media.spec.ts

```diff
@@ -456,7 +456,7 @@ describe('Media Metadata endpoints', () => {
   describe('get series by directory or filename', () => {
     it('should return series metadata', async() => {
       // this request populates the series metadata
-      let response = await got(`${appUrl}/api/media/seriestitle?title=Homeland S02E05`, { responseType: 'json' }) as UmsApiGotResponse;
+      let response = await got(`${appUrl}/api/media/seriestitle?title=Homeland S02E05&title=Homeland S02E05`, { responseType: 'json' }) as UmsApiGotResponse;
```

Comment:
Is this line intentional?

Reply:
Yeah, without the module, this test will fail, as the runtime code would be using the parameter when it is an array which doesn't work. This is a follow up to the tsc improvements and I typed all inputs as a string rather than an array